vignettes/polars.Rmd

```diff
@@ -51,88 +56,105 @@ dependencies other than R itself. This makes it very fast and lightweight to
 install, and so **polars** can immediately be used to tackle your big (or
 small!) data wrangling tasks.
 
-The rest of this vignette contains some simple examples to get you going. Note
-that the **polars** syntax and expressions are (deliberately) as close to the
-Python implementation as possible, so you can always refer to the
-[(py)polars book](https://pola-rs.github.io/polars-book/user-guide/) for more 
-ideas. Just remember to switch out any `.` (Python) for a `$` (R) when chaining
-methods. For example, here are two equivalent lines of code for some
-hypothetical dataset.
+## Documentation and tutorials
+
+Users can find detailed documentation for all objects, functions, and methods [on the Reference page of this website.](https://rpolars.github.io/reference/index.html) This documentation can also be accessed from the R console using the typical `?` syntax. For example, we will later use the `DataFrame()` constructor function and apply the `groupby()` method to a `DataFrame` object. The documentation for these can be accessed by typing these commands:
 
+```r
+?DataFrame
+?DataFrame_groupby
 ```
+
+The [Polars book](https://pola-rs.github.io/polars-book/user-guide/) offers a
+great introduction to the Polars data frame library, with a very large number
+of examples in Python and Rust. The syntax and expressions in the `polars`
+package for R are (deliberately) as close to the Python implementation as
+possible, so you can always refer to the [polars
+book](https://pola-rs.github.io/polars-book/user-guide/) for more ideas. Just
+remember to switch out any `.` (Python) for a `$` (R) when chaining methods.
+For example, here are two equivalent lines of code for some hypothetical
+dataset.
+
+```python
 # Python
-df.select([pl.col(["A", "B"]).sum()]))
+df.group_by("id").mean()
+```
 
+```r
 # R
-df$select(pl$col(c("A", "B"))$sum())
+df$groupby("id")$mean()
```

Comment:
Not saying you should revert, but my motivation for including two columns in the original example was to show the users would also need to switch over to R's native `c()` method (i.e., in addition to switching to `$`).

Reply:
Right that's useful info. But given that we haven't shown the user *anything* yet and it's their very first contact with **polars**, I feel that absolute simplicity and clarity serves our pitch better at this point. It's also not too hard to figure out `c()` in time when users try it out. That said, I don't feel extremely strongly about this, and could be swayed.

Reply:
Also, there is an example with `dat$select(pl$col(c("mpg", "hp")))` a little further down the page. So I think that's enough.